index.html

```diff
@@ -36,6 +36,116 @@ <h1>I’m Maaloum <br>
             </div>
         </section>
     </main>
+    <div class="grid-container">
+        <section class = "works">
+            <div class="job">
+                <img src="/images/Snapshoot-Portfolio.jpg" alt="job-1 image">
+                <div class="content">
+                    <h3>Tonic</h3>
+                    <ul>
+                        <li>Canopy</li>
+                        <li>Back End DEv</li>
+                        <li>2015</li>
+                    </ul>
+                    <p>A daily selection of privately <br> personalized reads; no accounts or sign-ups required.</p>
+                    <ul>
+                        <li><a href="#" class="programming">html</a></li>
+                        <li><a href="#" class="programming">css</a></li>
+                        <li><a href="#" class="programming">javascript</a></li>
+                    </ul>
+                    <button type="button">See Project</button>
```

Comment:
- [ ] I would recommend you work on the spacing and alignment of the elements in the project container as noted by the previous reviewer. Kindly make sure your design matches the Figma design chosen. Having a good UI adds quality to your project and also gives a good user experience to those that will be accessing your project, especially recruiters 😄 

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/177016056-b071bc0a-8dee-4c03-9322-4a2c640f876c.png) | ![image](https://user-images.githubusercontent.com/63473584/177016039-4329663f-3a6b-460a-85f6-3a392efb332c.png)  |